### ISSUE TITLE
Is the scaling position appropriate for the attention score?

### ISSUE THREAD

**Issue by High-East** [FrancescoSaverioZuppichini/ViT#2]
I think you need to multiply the scalings before calculate the attention scores.
Such like, 
att = F.softmax(energy * self.scaling, dim=-1)

But, You use this.
att = F.softmax(energy, dim=-1) * self.scaling

Is it right?

Please understand that English expression is awkward.

**Comment by FrancescoSaverioZuppichini** [comment]
Oh yeah, you are right. Thanks a lot!

**Comment by FrancescoSaverioZuppichini** [comment]
@High-East , done it - thanks :)
